rbac/ledger_sync/inbound/rbac_transactions.py

```diff
@@ -250,7 +237,7 @@ def add_sawtooth_prereqs(entry_id, inbound_entry, data_type):
         inbound_entry["object_type"] = addresser.ObjectType.ROLE.value
     else:
         raise ValueError(
-            "Expecting data_type to be 'user' or 'group, found: " + data_type
+            "Expecting data_type to be 'user' or 'group, found: %s" + data_type
```

Comment:
Small typo here as this will just print the actual %s seems we didn't added the `, data_type` at the end of the string.